modules/util/css-builder/build.gradle

```diff
@@ -45,6 +45,8 @@ dependencies {
 	compile group: "org.slf4j", name: "slf4j-log4j12", version: "1.7.2"
 	compile group: "org.sonatype.plexus", name: "plexus-build-api", transitive: false, version: "0.0.7"
 
+	compileInclude group: "com.beust", name: "jcommander", version: "1.72"
```

Comment:
We already have `compile` dependencies, so I don't see any reason to embed this particular jar and leave all the other ones as transitive dependencies in the `pom.xml` file.